client/src/components/AuthForm.tsx

```
@@ -0,0 +1,169 @@
+              alt="Auth Background"
+              fill
+              className="object-cover dark:brightness-[0.2] dark:grayscale"
+              quality={100}
+              priority
+            />
+          </div>
+        </CardContent>
+      </Card>
+
+      <div className="text-balance text-center text-xs text-muted-foreground [&_a]:underline [&_a]:underline-offset-4 hover:[&_a]:text-primary">
+        By clicking continue, you agree to our <a href="#">Terms of Service</a>{" "}
```

Biome (1.9.4)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)